### PR TITLE
Changes added for attachments to salesforce sobjects.

### PR DESCRIPTION
**Pull request by vikasrathee-cs**
PLUGIN-1539 - Changes done for adding attachments to salesforce sobjects.

### CODE REVIEW

File: docs/Salesforce-batchsink.md [hunk 1]
```diff
@@ -105,3 +105,21 @@ This value cannot be greater than 10,000,000.
...
+transform plugin called **Field Encoder** or encode directive in Wrangler plugin, which transforms byte data into a
+base64 string.
+
+Before running the pipeline, there are some file size constraints that you need to consider:
+
+- File Size Limit: The file size cannot exceed 10 MB, which is also the maximum size per batch.
+
+- Maximum Records per Batch: The number of records per batch cannot exceed 1000.
+
+Please ensure that the files you want to ingest comply with these constraints to avoid any issues during the pipeline
```
Comment (peanutbuttercrackers) [inline docs/Salesforce-batchsink.md]: Remove the word "Please". Public source of guideline: https://developers.google.com/style/word-list#please

File: docs/Salesforce-batchsink.md [hunk 2]
```diff
@@ -81,18 +81,18 @@ Below is a non-comprehensive list of **sObjects** that are not currently availab
 
 **Operation:** Operation used for writing data into Salesforce.  
 Insert - adds records.  
-Upsert - upserts the records. Salesforce will decide if sObjects 
+Upsert - upserts the records. Salesforce will decide if sObjects
 are the same using external ID field.  
 Update - updates existing records based on Id field.
```
Comment (peanutbuttercrackers) [inline docs/Salesforce-batchsink.md]: Here and elsewhere in this file, ID should be in all caps (not "Id" or "id"), unless it's code in text. Public source of guideline: https://developers.google.com/style/word-list#id

**Review by peanutbuttercrackers — COMMENTED** [review]
I'll incorporate this content in the Salesforce sink documentation on the CDAP site. Suggested a couple of tiny changes for Google Dev Style.

File: docs/Salesforce-batchsink.md (hunk 1 above)
Comment (vikasrathee-cs) [reply docs/Salesforce-batchsink.md]: done

File: docs/Salesforce-batchsink.md (hunk 2 above)
Comment (vikasrathee-cs) [reply docs/Salesforce-batchsink.md]: done

**Comment by MrRahulSharma** [comment]
Please add Unit Tests.

**Comment by vikasrathee-cs** [comment]
> Please add Unit Tests.

Added.

**Review by MrRahulSharma — APPROVED** [review]

**Pull request merged**